### ISSUE TITLE
Accessibility Issue: Repetitive reading & not focusable for navigation buttons (screen reader)

### ISSUE THREAD

**Issue by qqmmm** [Simon-Initiative/oli-torus#4476]
**Describe the bug**
When using the screen reader, the four navigation buttons on the left will be read as "image 'publish'" "image 'approve'", etc, creating additional cognitive overload for screen reader users. 
When turning off the screen reader and using the tab key to navigate through the page, only the four navigation buttons are not focusable. Sighted users might get lost while navigating. 

**To Reproduce**
Steps to reproduce repetitive reading:
1. use a screen reader (I used the Chrome extension screen reader for testing);
2. press "tab" key to navigate;
3. find "projects, create, publish, and improve"
4. you will hear "image 'projects' pop-up button collapse"

Steps to reproduce not focusable items:
1. use the tab key to navigate through the whole page
2. find "projects, create, publish, and improve"
3. you will see that there is no highlighted box surrounded these buttons while hovering on them. 

**Expected behavior**
Repetitive reading: the screen reader should read "'projects' pop-up button collapse"
set alt="", it will stop reading "image"

not focusable items: There should be a highlighted box on the button when using the tab key. 

**Screenshots**
![image](https://github.com/Simon-Initiative/oli-torus/assets/146996796/569fb7f9-e9cd-4ce8-b36b-6c0626af6dee)

![image](https://github.com/Simon-Initiative/oli-torus/assets/146996796/bf0261a0-4b71-4549-ba43-7de013e465b3)

**Environment (please complete the following information):**
 - Win
 - Browser: Chrome
 - Version: [e.g. 22]

**Additional context**
Add any other context about the problem here, for instance a full link to the page or resource if this problem is specific to a course or a resource in a course.
https://proton.oli.cmu.edu/authoring/project/test_course_basic_words_in_oth/curriculum
